docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -18,23 +19,28 @@ quotas using either the WebUI or CLI.
 
 #. As an admin user, either create a new workspace or edit an existing workspace.
 
-#. In the "Namespace Bindings" section, type a name for the namespace or leave it blank. If left
-   blank, the workspace will be bound to the default namespace configured in the Master
-   Configuration YAML file.
+#. In the "Namespace Bindings" section, type a name for the namespace or leave it blank. The
+   behavior depends on the master configuration. For example, if the ``default_namespace`` is
+   defined, leaving the namespace blank binds the workspace to the "default" namespace (even if it
+   is not set) configured for that Kubernetes cluster. If a different namespace name is provided,
+   the workspace is bound to that namespace.
```

Comment:
In the "Namespace Bindings" section, enter a name for the namespace or leave it blank. If you specify a namespace in the namespace field, the workspace is bound to that namespace. If the field is left blank, the workspace is bound to the namespace specified in the ``resource_manager.default_namespace`` section of your master configuration when this configuration field is set, and is otherwise bound to the Kubernetes namespace, "default".